src/schedule/kat_key_schedule.rs

```diff
@@ -49,7 +50,7 @@ struct Epoch {
 }
 
 #[derive(Serialize, Deserialize, Debug, Clone, Default)]
-struct KeyScheduleTestVector {
+pub struct KeyScheduleTestVector {
```

Comment:
Why do you need this and some of the other functions in here public? This module isn't public anywhere afaik such that this doesn't make any difference.